recipe/meta.yaml

```diff
@@ -7,87 +7,62 @@ package:
 
 source:
   url: https://pypi.io/packages/source/r/rpy2/rpy2-{{ version }}.tar.gz
-  sha256: be57f741d0c284b5d8785ab03dff0e829303e5ac30e548d5ceb46e05b168812e
-  patches:
-    # patches to be upstreamed:
```

Comment:
Not sure if these were upstreamed or not but I got a successful build on Linux without them. Hopefully Windows and mscOS will be alright too.